src/components/Theme/constants.scss

```diff
@@ -126,6 +126,13 @@ $color-camera-button-disabled: #2b2d32;
   right: 0;
 }
 
+%overflow-drop-shadow {
+  @media (--shorter-viewport) {
+    padding-top: 8 * $unit;
+    box-shadow: 0 -5px 10px -5px #7b7b7b;
```

Comment:
```suggestion
    background-color: white;
    bottom: 0;
    box-shadow: 0px 0px 8px 2px black;
    content: '';
    left: -1em;
    position: absolute;
    right: -1em;
    top: -8 * $unit;
    z-index: 0;

    & + button {
      z-index: 1;
    }
```

Reply:
Nice. Though I probably would make the `background-color` inherited since integrators might set their own background colour for the main container. Will check it out locally.